lab-david/server.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const express = require('express');
+const debug = require('debug')('cfgram:server');
+const dotenv = require('dotenv');
+const mongoose = require('mongoose');
+const cors = require('cors');
+const morgan = require('morgan');
+
+const authRouter = require('./route/auth-router.js');
+const photoAlbumRouter = require('./route/photoalbum-router.js');
+const photoRouter = require('./route/photo-router.js');
+const errors = require('./lib/error-middleware');
+
+dotenv.load();
```

Comment:
you can do this on the import without storing it into a var.

`require('dotenv').load()`